src/libs/dps-sftp-starter/src/test/java/SftpServiceImplTest.java

```diff
@@ -82,4 +86,9 @@ public void withSftpExceptionShouldThrowDpsSftpException() {
         });
     }
 
+    @Test
+    public void withValidFileShouldMove() {
+        sut.moveFile(FILE_1, FILE_2);
```

Comment:
you can use something like ```assertions.doNotThrows(() -> { ... })``` this add an assertion to the test to make sure no exceptions are thrown.